pkg/network/dns_snooper.go

```diff
@@ -232,9 +231,10 @@ func (s *SocketFilterSnooper) getCachedTranslation() *translation {
 
 	// Recycle buffer if necessary
 	if t.ips == nil || len(t.ips) > maxIPBufferSize {
-		t.ips = make([]util.Address, 30)
+		t.ips = make(map[util.Address]time.Time, 30)
```

Comment:
I don't understand this. Are we trying to resize the map and then copy over the entries?

Reply:
The `translation` object is allocated once and reused. This function is called to obtain the object for use, so we need to ensure it is "cleaned" up.